docs/global.rst.inc

```diff
@@ -1,6 +1,5 @@
 .. highlight:: bash
-.. |project_name| replace:: Borg-Import
-.. |git_url| replace:: https://github.com/borgbackup/borg-import.git
+.. _repo replace:: https://github.com/borgbackup/borg-import.git
```

Comment:
maybe rather use _githubrepo considering borg also talks about (borg, not git) repos.